src/App.module.scss

```diff
@@ -23,16 +23,22 @@ ol, ul {
 }
 
 a {
+  transition: .3s all;
```

Comment:
move it to constant. The animation time have to the same in project.

Reply:
fixed